1-exercises/D-for-loop/exercise.js

```diff
@@ -26,6 +26,14 @@ const AGES = [
     49
 ];
 
+AGES.forEach(k => {
+    WRITERS.forEach(i => {
+        console.log(i +  " is " + k + " years old")
+    })
+})
```

Comment:
This doesn't look correct. Currently, it will say that every writer is 59,40,41,63 and 49 years old.